src/topology/algebra/uniform_ring.lean

```diff
@@ -150,8 +151,45 @@ instance : comm_ring (completion R) :=
       (assume a b, by rw [← coe_mul, ← coe_mul, mul_comm]),
  ..completion.ring }
 
-end uniform_space.completion
+namespace ring_completion
```

Comment:
This shouldn't need a new namespace, a ring completion is just a regular completion, just like a group homomorphism is just a monoid homomorphism.